tools/mesonify.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/python3
+
```

Comment:
A small docstring would be nice to explain what the tool does.

Can this be used to update the existing `meson.build` files?

Reply:
Also, it would be nice to have this file properly formated.

Reply:
No, this tool was just used to generate empty meson tree, which requires manual tuning.
I updated it to prevent any harm (i.e. don't overwrite hand-crafted `meson.build` files).